lib/Twig/Lexer.php

```diff
@@ -136,7 +136,7 @@ public function tokenize($code, $filename = null)
             mb_internal_encoding($mbEncoding);
         }
 
-        return new Twig_TokenStream($this->tokens, $this->filename, $this->env->isDebug() ? $code : '');
+        return new Twig_TokenStream($this->tokens, $this->filename, $code);
```

Comment:
I would revert this totally and remove the `$code` from `Twig_TokenStream` (this is purely internal and nobody should rely on this).